contracts/PermsMng.sol

```diff
@@ -0,0 +1,87 @@
+pragma solidity 0.4.25;
+
+contract PermsMng {
+    // owner from entry contract
+    address owner;
+    // perm addresses
+    address[33] public perms;
```

Comment:
why not `address[]`

Reply:
it threw an error "Instruction is not supported."